src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1213,20 +1213,8 @@ static bool addTriggeredSubscriptions_withCache
     {
       LM_T(LmtMongo, ("adding subscription: '%s'", sub.toString().c_str()));
 
-      long long throttling       = sub.hasField(CSUB_THROTTLING) ? getField(sub, CSUB_THROTTLING).numberLong() : -1;
-      long long lastNotification = -1;
```

Comment:
Probably this PR deserver a line like this in CNR:

"Hardening: safer read of lastNotification, expiration and throttling fields in csub collection, assuming that the field can be either int or long (always written as long)"

or something similar.